auth/README.md

```diff
@@ -35,9 +35,18 @@ Getting Started
  1. Run `firebase use --add` and select your Firebase project.
 
 To run the sample app locally during development:
- 1. Run `firebase serve`.
-    This will start a server locally that serves `auth/index.html` on `http://localhost:<port>`. Check the output of the command for the exact port.
- 1. Navigate in your browser to the URL output by the `firebase serve` command.
+ 1. Run `npm install` to install dependencies.
+ 1. Run `firebase emulators:start` to start the local Firebase emulators. Note: phone authentication required ReCaptcha verification which does not work with the Firebase emulators. These examples skip connecting to the emulators.
+ 1. Run `npm run dev` to serve the app locally using Vite
+    This will start a server locally that serves `auth/index.html` on `http://localhost:5173/auth/index.html`. 
+
+
+Running the app using the Firebase CLI:
+ 1. Alternatively, you can also serve the app via the Firebase CLI.
+ 1. Run `npm run build` to build the app using Vite.
+ 1. Run `firebase emulators:start` to start the local Firebase emulators. Note: phone authentication required ReCaptcha verification which does not work with the Firebase emulators. These examples skip connecting to the emulators.
```

Comment:
```suggestion
 1. Run `firebase emulators:start` to start the local Firebase emulators. Certain features that require Recaptcha are run in production environment instead of local emulators.
```